install-required-packages.sh

```
@@ -641,9 +641,10 @@ declare -A pkg_python3_pip=(
 declare -A pkg_python3_yaml=(
 	   ['arch']="python-yaml"
 	 ['centos']="python3-PyYAML"
+	 ['centos']="ERROR/I don't know how to install PyYAML library for python3"
 	 ['debian']="python3-yaml"
 	 ['gentoo']="dev-python/pyyaml"
-	   ['rhel']="python3-PyYAML"
+	   ['rhel']="ERROR/I don't know how to install PyYAML library for python3"
```

This should throw an error.